app/services/feeds/feed_entry_presenter.rb

```diff
@@ -0,0 +1,41 @@
+module Feeds
+  class FeedEntryPresenter
```

Comment:
I think this adds some app confusion having this class here within services when there is already a presenters directory for the other ones in this app

Reply:
Do you think it's better to put them back into their directories without module, or do you see another option?